### PR TITLE
Ci/adding nanos+

### PR DESCRIPTION
**Pull request by lpascal-ledger**

### CODE REVIEW

**Comment by lpascal-ledger** [comment]
[Speculos 'bug' due to new version of Werkzeug](https://github.com/LedgerHQ/speculos/issues/282) breaks the tests. Waiting for the new version.

File: .github/workflows/ci-workflow.yml [hunk 1]
```diff
@@ -53,10 +54,30 @@ jobs:
           name: boilerplate-app-nanoX-debug
           path: bin
 
+  job_nanoSP_build_debug:
```
Comment (fbeutin-ledger) [inline .github/workflows/ci-workflow.yml:57]: I suggest using a matrix strategy to reduce code duplication. A simple use is done in the review for dynamic exchange app build

Comment (lpascal-ledger) [reply .github/workflows/ci-workflow.yml]: Nice, let's see.

File: .github/workflows/ci-workflow.yml [hunk 2]
```diff
@@ -12,67 +12,31 @@ on:
       - develop
 
 jobs:
-  job_nanoS_build_debug:
-    name: Build debug app for NanoS
+  nano_build:
+    name: Build boilerplate application for NanoS, X and S+
+    strategy:
+      matrix:
```
Comment (jibeee) [inline .github/workflows/ci-workflow.yml:18]: I didn't know `matrix`. Excellent change.

**Review by jibeee — APPROVED** [review]

**Pull request merged**